xline/src/server/lease_server.rs

```diff
@@ -170,6 +170,7 @@ where
                 debug!("Receive LeaseKeepAliveRequest {:?}", keep_alive_req);
                 // TODO wait applied index
```

Comment:
If we have `wait_synced` method here, shall we keep this TODO? @themanforfree. This TODO comment is introduced in the commit 42e85d61b1496592ff4bf358331123a17b1634f2.

Reply:
I think we can remove it here

Reply:
done